### ISSUE TITLE
Support for More Wii Retail Disc Titles containing emulated ROMs

### ISSUE THREAD

**Issue by PW5190** [farmerbb/RED-Project#59]
Hey, I have a question. I know you can extract the ROMs for the Masterpieces from the _Super Smash Bros. Brawl_ disc, but I want to know if you can do the same with other Wii disc games containing emulated ROMs, such as _Super Mario All-Stars_ (which is actually the entire SNES ROM on a Wii disc) and _Kirby's Dream Collection_ (which contains 1 NES, 1 N64, 2 GB, and 2 SNES ROMs on one disc). If ROM extraction from two of these games are possible, let me know. Also, if you find any more Wii disc game compilations which use ROMs in emulation, keep me posted.

Notice: This issue only covers the retail Wii disc games, not downloadable Virtual Console or WiiWare titles.

**Comment by einstein95** [comment]
Yes, you can. For All Stars, the files on the disc are exactly the same as
in Wii VC titles, just not in a wad file.

**Comment by kjetil-f** [comment]
I followed this when I extracted All-Stars: https://www.retrogameboards.com/t/the-ripping-thread-how-to-build-your-own-legit-retro-rom-library/98
